sycl/test/on-device/basic_tests/interop/get_native_ze.cpp

```diff
@@ -0,0 +1,35 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.ze.out
+// RUN: env SYCL_DEVICE_FILTER="level_zero" %t.ze.out
```

Comment:
Should there also be OpenCL version of the test?

Reply:
Sure, why not